jetty-io/src/main/java/org/eclipse/jetty/io/AbstractEndPoint.java

```diff
@@ -175,7 +175,7 @@ public final void close()
         close(null);
     }
 
-    protected final void close(Throwable failure)
+    public final void close(Throwable failure)
```

Comment:
I'm OK with this change, but I think we should probably take the opportunity to change close to always take a throwable throughout the code base.  I'll prepare a PR or patch to extend this... which will need further review.